Makefile

```diff
@@ -77,8 +77,8 @@ LOCAL_EDX_PINS = requirements/edx-platform-constraints.txt
 PLATFORM_BASE_REQS = https://raw.githubusercontent.com/edx/edx-platform/master/requirements/edx/base.txt
 
 check_pins: ## check that our local copy of edx-platform pins is accurate
-	echo "### DON'T edit this file, it's copied from edx-platform. See make upgrade" > $(LOCAL_EDX_PINS)
-	curl -fsSL $(PLATFORM_BASE_REQS) | grep -v '^-e' | grep -v 'via edx-enterprise$$' >> $(LOCAL_EDX_PINS)
+ 	#echo "### DON'T edit this file, it's copied from edx-platform. See make upgrade" > $(LOCAL_EDX_PINS)
```

Comment:
?

Reply:
This would cause packages other than celery to upgrade too which are pinned in edx-platform, you can just write a command that removes celery from this file in after we copy this file from edx-platform.